presto-main/src/main/java/com/facebook/presto/sql/planner/SubqueryPlanner.java

```diff
@@ -230,6 +233,13 @@ private PlanBuilder appendExistSubqueryApplyNode(PlanBuilder subPlan, ExistsPred
 
         PlanNode subqueryPlan = createRelationPlan(existsPredicate.getSubquery()).getRoot();
 
+        Symbol exists = symbolAllocator.newSymbol("exists", BOOLEAN);
+        if (isAggregation(subqueryPlan)) {
+            // aggregation always return at least one row
```

Comment:
This is not necessarily true. If it has a GROUP BY on anything other than (), it will return 0 rows if the input contains 0 rows:

```sql
SELECT count(*) 
FROM (VALUES 1,2,3,4) t(x) 
WHERE false 
GROUP BY x;
```

=>

```
 _col0
-------
(0 rows)
```